LayoutTests/accessibility/animated-dropdown.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+<script src="resources/jquery-3.6.1.js"></script>
+<script src="../resources/accessibility-helper.js"></script>
+<script src="../resources/js-test.js"></script>
+</head>
+<body>
+<main id="main">
+  <section>
+	  <button id="button-one" class="dropdown">Expand first</button>
+	  <div style="display:none">This text begins as hidden</div>
+  </section>
+  <section><button>Foo</button></section>
+</main>
+<footer><a href="#url">apple.com</a></footer>
+<script>
+var output = "This test ensures the accessibility tree is correct after an animated dropdown is opened via button press.\n\n";
+
+if (window.accessibilityController) {
+    window.jsTestIsAsync = true;
+
+    output += "First traversal:\n";
+    output += dumpAXSearchTraversal(accessibilityController.rootElement.childAtIndex(0));
+    var secondTraversal;
+    setTimeout(async function() {
+        await sleep(1000)
```

Comment:
Why do we need to sleep here?

Reply:
Oops, I think this is left over from when I was tweaking this test. Even if a sleep is necessary, surely it can be shorter than one second while still triggering the bug-under-test. Let me go back and look into this.

Reply:
The `sleep` can be removed outright without affecting whether the bug is triggered, so I've done so in the latest commit.